Tests/RudderStackAnalyticsTests/Plugins/DeviceMode/CustomIntegrationPluginTests.swift

```diff
@@ -10,11 +10,19 @@ import Testing
 
 struct CustomIntegrationPluginTests {
     
+    var analytics: Analytics
+    
+    init() {
+        let mockConfiguration = MockProvider.createMockConfiguration()
+        mockConfiguration.flushPolicies = []
+        
+        self.analytics = Analytics(configuration: mockConfiguration)
+    }
```

Comment:
Since the nature of this change is different, it could have been made in a new PR to reduce the no. of files changed. Please consider that in future.